chapters/operatorsandexpressions.tex

```diff
@@ -514,13 +511,16 @@ \subsection{Numeric Functions and Conversion Functions}\label{numeric-functions-
   \lstinline!Boolean leftJustified = true!: If true, the converted result is left justified in the string; if false it is right justified in the string.
 \item
   \lstinline!Integer significantDigits = 6!: Number of significant digits in the result string.
+  Only allowed when formatting a \lstinline!Real! value (and \lstinline!format! is not provided).
```

Comment:
The last part about format applies to all of the options, so I think it should either be removed (it's already clear from the synopsis that using format is a different overload that doesn't take options) or clarified above the option list so it applies to all of them.